apps/core/scenes/search/PlacePicker/PlacePickerRefetchContainer.js

```diff
@@ -0,0 +1,63 @@
+// @flow
+
+// This rule is disabled because this is just temporary the PlacePicker component.
+/* eslint-disable relay/unused-fields */
+
+import {
+  graphql,
+  createRefetchContainer,
+  type RelayRefetchProp,
+} from '@kiwicom/margarita-relay';
+import { debounce } from '@kiwicom/margarita-utils';
```

Comment:
Why don't you import `lodash.debounce` directly here?

Reply:
Nothing special but utils looks like a good place for it, and if we'll decide change lodash debounce implementation to something else it will be easy, change it just here, of cores the interface must be same.